arangod/Pregel/GraphStore.cpp

```diff
@@ -100,7 +99,7 @@ std::map<CollectionID, std::vector<VertexShardInfo>> GraphStore<V, E>::_allocate
 
   std::map<CollectionID, std::vector<VertexShardInfo>> result;
 
-  LOG_TOPIC(DEBUG, Logger::PREGEL) << "Allocating memory";
+  LOG_TOPIC(INFO, Logger::PREGEL) << "Allocating memory";
```

Comment:
I think this should stay at `DEBUG`?
```suggestion
  LOG_TOPIC(DEBUG, Logger::PREGEL) << "Allocating memory";
```